### PR TITLE
Change Dark Mode Contrast Search Box

### PR DESCRIPTION
**Pull request by ethanjoby**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes:https://github.com/cpinitiative/usaco-guide/issues/4152
Before:
Screenshot 2024-02-03 at 9 53 18 AM
After
Screenshot 2024-02-03 at 9 52 42 AM
4-7796-44dc-9703-ac7389a75b98">

### CODE REVIEW

File: src/components/ProblemsPage/SearchBox.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default function SearchBox(props: UseSearchBoxProps): JSX.Element {
       </div>
       <input
         id="search"
-        className="block w-full pl-12 pr-3 py-3 rounded-md bg-blue-100 dark:bg-blue-800 placeholder-gray-900 dark:placeholder-blue-300 focus:outline-none focus:border-blue-300 focus:shadow-outline-blue transition text-black dark:text-white"
+        className="block w-full pl-12 pr-3 py-3 rounded-md bg-blue-100 dark:bg-blue-100 placeholder-gray-900 dark:placeholder-blue-900 focus:outline-none focus:border-blue-300 focus:shadow-outline-blue transition text-black dark:text-white"
```
Comment (thecodingwizard) [inline src/components/ProblemsPage/SearchBox.tsx:39]: nice! can you also change the text color in dark mode? also, what are your thoughts on making the background a bit darker in dark mode, maybe `bg-blue-200`?

Thanks for making this change!

**Comment by mjh316** [comment]
@ethanjoby I saw that you applied for CPI Web Dev - how's this PR going?

File: src/components/ProblemsPage/SearchBox.tsx (hunk 1 above)
Comment (ethanjoby) [reply src/components/ProblemsPage/SearchBox.tsx:39]: Which text specifically? And to what color? Also for the background color is it the one inside the search box or outside?

**Comment by ethanjoby** [comment]
> @ethanjoby I saw that you applied for CPI Web Dev - how's this PR going?

Is there any other changes i need to make or is this good enough?

File: src/components/ProblemsPage/SearchBox.tsx (hunk 1 above)
Comment (ethanjoby) [reply src/components/ProblemsPage/SearchBox.tsx:39]: which text specifically and what color? Also should I change the outside background or searchbox background. Thanks!

Comment (thecodingwizard) [reply src/components/ProblemsPage/SearchBox.tsx:39]: the text inside the search box. Up to you which color -- it's just white on light background right now which is not good.

image

I was referring to the background of the input box.

Comment (ethanjoby) [reply src/components/ProblemsPage/SearchBox.tsx:39]: i made another pull request

**Comment by danielzsh** [comment]
Closing this since another pr was opened

**Pull request closed without merging**